com.github.franjegueje.deckico.yml

```diff
@@ -0,0 +1,32 @@
+app-id: com.github.franjegueje.deckico
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: deckico-gui
+finish-args:
+  - --socket=x11
+  - --filesystem=~/.config/autostart
+  - --filesystem=~/.local/share/Steam/steamapps/compatdata
+  - --filesystem=~/.local/share/Steam/steamapps/shadercache
+  - --filesystem=~/.local/share/Steam/userdata:ro
+  - --filesystem=/run/media
+modules:
+  - name: deckico
+    buildsystem: simple
+    build-commands:
+      - install -D deckico-gui /app/bin/deckico-gui
+      - install -D deckico-cli /app/bin/deckico-cli
+      - install -D yad /app/bin/yad
+      - mkdir -p /app/usr/share/icons/deckico
+      - cp -Rf images/* /app/usr/share/icons/deckico/.
+    sources:
+      - type: file
+        path: deckico-gui
+      - type: file
+        path: deckico-cli
+      - type: file
+        path: yad
```

Comment:
1. these files are not found
2. what they seem to look like is that they wouldn't belong here.

You should either build from source (preferred) which mean pulling upstream sources here, or use a prebuilt binary.

Reply:
I have added the files